File: 백준/Bronze/32184. 디미고에 가고 싶어！/디미고에 가고 싶어！.py
start, end = map(int, input().split())
page = end-start+1
shot = 0

if page%2 == 0:
  if start%2 == 0:
    shot += (page-2)//2+2
  else:
    shot += page//2
else:
    shot += (page-1)//2+1

print(shot)

File: 백준/Bronze/32184. 디미고에 가고 싶어！/디미고에 가고 싶어！.cc
#include <iostream>
using namespace std;

int main() {
  int start, end;
  cin >> start >> end;

  int page = end - start + 1;
  int shot = 0;

  if(page%2 == 0) {
    if(start%2 == 0) shot += (page-2)/2+2;
    else shot += page/2;
  }
  else shot += (page-1)/2+1; 

  cout << shot << endl;

  return 0;
}